<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1306</width>
    <height>996</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>SortedLocationsModel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="treeViewSortedLocations">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="4">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>CustomConfigsModel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="listViewCustomConfig"/>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>FavoriteModel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="listViewFavorite"/>
       </item>
      </layout>
     </item>
     <item row="0" column="3">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>StaticIpsModel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="listViewStaticIps"/>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>LocationsModel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="treeViewLocations">
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>WidgetLocations</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>WidgetCities</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="3">
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Favorites</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QComboBox" name="cbSort">
         <item>
          <property name="text">
           <string>Geography</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Alphabet</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Latency</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOriginal">
         <property name="text">
          <string>Original</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDeletedLocations">
         <property name="text">
          <string>Deleted locations</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDeletedCities">
         <property name="text">
          <string>Deleted cities</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnChangedLocations">
         <property name="text">
          <string>Changed locations captions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnChangedCities">
         <property name="text">
          <string>Changed cities captions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnChangedLocationsOrder">
         <property name="text">
          <string>Changed locations order</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnChangedCitiesOrder">
         <property name="text">
          <string>Changed cities order</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClipboard">
         <property name="text">
          <string>Update from clipboard</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCustomConfig1">
         <property name="text">
          <string>Custom config 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCustomConfig2">
         <property name="text">
          <string>Custom config 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClearCustomConfig">
         <property name="text">
          <string>Clear custom config</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBestLocation">
         <property name="text">
          <string>Random best location</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSimple1">
         <property name="text">
          <string>Simple1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRandom">
         <property name="text">
          <string>Random</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
